crates/bench/benches/special.rs

```diff
@@ -48,6 +50,18 @@ fn custom_module_benchmarks(c: &mut Criterion) {
     }
 }
 
+fn game_sql_benchmarks(c: &mut Criterion) {
```

Comment:
I think we should just add the utility for right now. We can decide later if we want this to be a CI benchmark.